swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -905,7 +1003,7 @@ private void updateRowset() {
 			logger.debug(() -> String.format("%s: Setting to null.", getColumnForLog()));
 			setBoundColumnText(null);
 		} else {
-			logger.debug(String.format("%s: Setting to %s.",  getColumnForLog(), mapping));
+			logger.debug(() -> String.format("%s: Setting to %s.",  getColumnForLog(), mapping));
```

Comment:
This is where the logger.debug "() -> " fix needed to be after logic was moved out of the action listener. Again, I think that's the worst of it.